scripts/utils/formatElmCompilerErrors.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var chalk = require('chalk');
+var ctx = new chalk.constructor({ enabled: true });
+var error = ctx.bold.red;
+var filename = ctx.cyan;
+var isBrowser = typeof window === 'object';
+
+if (isBrowser) {
+  ctx.level = 1;
+  error = ctx.cyan;
+  filename = ctx.green;
+}
+
+function stripRedundantInfo(error) {
+  return error.replace(
+    /Module build failed: Error: Compiler process exited with error Compilation failed/g,
+    ''
+  );
+}
+
+module.exports = function formatElmCompilerErrors(messages) {
+  var errors = messages.errors;
+  var warnings = messages.warnings;
+  return errors.length > 0
+    ? {
+        errors: errors
+          .map(x =>
```

Comment:
I shouldn't be using arrow functions here because of browser compatibility issues.